pkg/cri/cri.go

```diff
@@ -97,8 +99,12 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create containerd client")
 	}
+	pf, err := getPortForwardService(ic)
```

Comment:
Did we need to figure out a way to make this per runtime?

Reply:
My intention was to allow shims to implement port forward. I don't think we had a need to implement it per "runtime" (i.e. v1, v2, v2/shim). My intention was for 'pf' to return an error in cases where it wasn't supported and fall back to the previous logic. I do that around line 75 in sandbox_portforward_linux.go.

I'm open to thinking about it but I can't really imagine how it would work unless we push the code in localPortForward in sandbox_portforward_linux.go into runtime/v2 somewhere. I don't think we really want to do that.